QuizProject/Game/ViewModels/NormalGameViewModel.swift

```diff
@@ -62,4 +65,21 @@ final class NormalGameViewModel: ObservableObject {
             currentQuestion += 1
         }
     }
+    
+    func addTime() {
+        seconds[currentQuestion] += 20
+        points -= 2
+        isAddTime[currentQuestion].toggle()
+    }
+    
+    func excludeAnswers(){
+        while (questions[currentQuestion].answers.count != 2){
```

Comment:
Would that crash if it picks 3 the second time?

Reply:
Yes, and I have fixed it up in the next commit.